edn/src/types.rs

```diff
@@ -0,0 +1,73 @@
+
+use std::collections::{BTreeSet, BTreeMap, LinkedList};
+use std::cmp::{Ordering, Ord, PartialOrd};
+use ordered_float::OrderedFloat;
+
+/// We're using BTree{Set, Map} rather than Hash{Set, Map} because the BTree variants implement Hash
```

Comment:
I agree (and I've made it better) That said, we don't really have a defined API yet, so I'm OK with better docs being lower priority.